tomls/markets/spot/eth.toml

```diff
@@ -38,7 +38,8 @@ func = "updatePriceData"
 args = [
     "<%= extras.synth_eth_market_id %>",
     "<%= extras.eth_oracle_id %>",
-    "<%= extras.eth_oracle_id %>"
+    "<%= extras.eth_oracle_id %>",
```

Comment:
This change affects other deployments that rely on older version of spot market.

```js
Failure on step invoke.setSynthPriceEth
Error: too many arguments: passed to contract (count=5, expectedCount=3, code=UNEXPECTED_ARGUMENT, version=contracts/5.7.0)
```

Reply:
We need to have this change separated from common configs and specify for andromeda deployment exclusively.